examples/server.rs

```diff
@@ -13,8 +13,8 @@ fn main() {
     let port = "7878";
 
     let server = Server::new(|request, mut response| {
-        info!("Request received. {} {}", request.method(), request.uri());
-        Ok(response.body("Hello Rust!".as_bytes())?)
+        let msg = format!("The path you requested was: '{}'", request.uri().path());
```

Comment:
this changes the behavior of the example; I'd prefer to keep it the same.

Reply:
sorry, forgot I had changed that when I was doing a quick check of this functionality

Reply:
agree. the example staying the same is like a test passing, IMO.